src/components/startupDelay.js

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react';
+import { ProgressBar } from 'react-mdl';
+import getConfig from '../utils/appConfig';
+import App from '../App';
+
+export default class handleClusterDelay extends Component {
+    state = {
+        minutes: 5,
+        seconds: 0,
+        status:"Please wait for For clusters to Spin up",
+        waitTime:"Estimate Wait Time",
+        clusterUrl:""
+      }
+      setTimer(){
+            if (this.state.seconds > 0){
+                this.setState({seconds:this.state.seconds-1})
+            }else if (this.state.seconds === 0) {
+                        if (this.state.minutes === 0) {
+                          this.setState({status:"Error in Loading custers",waitTime:""})
+                          this.componentWillUnmount()
+                          return;
+                        } else {
+                        this.setState({minutes:this.state.minutes-1,seconds: 59})
+                        }   
+            }
+      }
+      componentDidMount() {
+              this.timer = setInterval(()=> this.getClusterUrl(), 10000); 
+              this.timer = setInterval(()=> this.setTimer(), 1000);
+        }
+      componentWillUnmount() {
+        this.timer = null; 
+      }
+      async getClusterUrl(){
+        let that = this;
+       await fetch(getConfig("REACT_APP_OS_URL"), {method:'POST',mode:'cors',cache:'no-cache',headers:{"Content-Type": "application/json"}})
```

Comment:
The "REACT_APP_OS_URL" will return something like `https://osdnsname.tld` so you will need to add the route yourself
```suggestion
       await fetch(`${getConfig("REACT_APP_OS_URL")}/connect`, {method:'POST',mode:'cors',cache:'no-cache',headers:{"Content-Type": "application/json"}})
```